packages/broker/test/integration/plugins/storage/Storage.test.ts

```diff
@@ -265,59 +310,100 @@ describe('Storage', () => {
         expect(results3).toEqual([])
     }, 20000)
 
-    test('fast big stream', async () => {
+    describe('fast big stream', () => {
+        let storedStreamId: string
+        const NUM_MESSAGES = 1000
+        const MESSAGE_SIZE = 1e3 // 1k
+
+        beforeEach(async () => {
```

Comment:
Could use `beforeAll` here?

Reply:
Tried that but `beforeAll` seems to run before any of the other `beforeEach` calls, so client etc aren't set up.